presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -183,6 +177,48 @@ fun EventScreen(
         }
     )
 
+    if (bottomSheetState.isVisible) {
```

Comment:
You are only showing the bottomsheet, when it's visible
You should always show the bottomsheet, and handle it's visibility using the ".hide()" and ".show()"

If you remove th if check + you do what I commented before about the initial state, It should work fine